### PR TITLE
util.StoreAccount() take address.Address as input 2155

### PR DESCRIPTION
**Pull request by lzxm160**
work on #2155

### CODE REVIEW

File: action/protocol/account/transfer.go [hunk 1]
```diff
@@ -110,9 +110,12 @@ func (p *Protocol) handleTransfer(ctx context.Context, act action.Action, sm pro
 	if err := recipient.AddBalance(tsf.Amount()); err != nil {
 		return nil, errors.Wrapf(err, "failed to update the Balance of recipient %s", tsf.Recipient())
 	}
-
+	addr, err := address.FromString(tsf.Recipient())
```
Comment (dustinxie) [inline action/protocol/account/transfer.go:113]: use recipientAddr above

File: action/protocol/poll/util.go [hunk 2]
```diff
@@ -178,7 +180,11 @@ func setCandidates(
 				return err
 			}
 		}
-		if err := accountutil.StoreAccount(sm, candidate.Address, delegate); err != nil {
+		addr, err := address.FromString(candidate.Address)
```
Comment (dustinxie) [inline action/protocol/poll/util.go:183]: name it candAddr

File: action/protocol/account/transfer.go (hunk 1 above)
Comment (lzxm160) [reply action/protocol/account/transfer.go:113]: sure

File: action/protocol/account/transfer.go [hunk 3]
```diff
@@ -110,9 +110,12 @@ func (p *Protocol) handleTransfer(ctx context.Context, act action.Action, sm pro
 	if err := recipient.AddBalance(tsf.Amount()); err != nil {
 		return nil, errors.Wrapf(err, "failed to update the Balance of recipient %s", tsf.Recipient())
 	}
-
+	recipientAddr, err = address.FromString(tsf.Recipient())
```
Comment (dustinxie) [inline action/protocol/account/transfer.go:113]: i mean, line 67 already has this code, directly use it

Comment (dustinxie) [reply action/protocol/account/transfer.go:113]: no need to call FromString() again

**Pull request merged**